orcid-core/src/main/java/org/orcid/core/manager/impl/WorkManagerImpl.java

```diff
@@ -222,8 +222,8 @@ private void setIncomingWorkPrivacy(WorkEntity workEntity, ProfileEntity profile
     @Override
     @Cacheable(value = "works-summaries", key = "#orcid.concat('-').concat(#lastModified)")
     public List<WorkSummary> getWorksSummaryList(String orcid, long lastModified) {
-        List<MinimizedWorkEntity> works = workDao.findWorks(orcid);        
-        return jpaJaxbWorkAdapter.toWorkSummaryFromMinimized(works);
```

Comment:
@rcpeters It has those fields. I missed the extended BaseEntity. Reverted and updated just the Mapper.